src/main/scala/uk/gov/homeoffice/drt/IEIDashboardServer.scala

```diff
@@ -0,0 +1,49 @@
+package uk.gov.homeoffice.drt
+
+import cats.effect.{ConcurrentEffect, ContextShift, Timer}
+import cats.implicits._
+import fs2.Stream
+import org.http4s.client.blaze.BlazeClientBuilder
+import org.http4s.implicits._
+import org.http4s.server.blaze.BlazeServerBuilder
+import org.http4s.server.middleware.Logger
+import uk.gov.homeoffice.drt.api.{ArrivalRoutes, PublicRoutes}
+import uk.gov.homeoffice.drt.applicative.ArrivalFlights
+import uk.gov.homeoffice.drt.repository.ArrivalRepository
+import uk.gov.homeoffice.drt.service.{AirlineService, ArrivalService}
+
+import scala.concurrent.ExecutionContext.global
+
+object IEIDashboardServer {
+  def stream[F[_] : ConcurrentEffect](cfg: Config)(implicit T: Timer[F], C: ContextShift[F]): Stream[F, Nothing] = {
+    for {
+      _ <- BlazeClientBuilder[F](global).stream
+
+      session = ResourceObject.session(cfg.database)
+
+      clientResource = BlazeClientBuilder[F](global).resource
+      airlineService = new AirlineService(clientResource)
+//      _ = ResourceObject.updateAirLines(cfg.airline,airlineService)
```

Comment:
Shall we remove this comment?

Reply:
I will remove it but kept in case not with line next to it which uses hardcode carrierName . but I guess as value of Carrier Name doesn't change and mostly static so hardcore should be fine .